pkg/option/option_test.go

```diff
@@ -150,6 +155,17 @@ func TestOption(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "pcidb",
+			opts: []*option.Option{
+				option.WithPCIDB(pcidb),
+				option.WithChroot("/my/chroot/dir"),
+			},
+			merged: &option.Option{
+				Chroot: stringPtr("/my/chroot/dir"),
+				PCIDB:  pcidb,
+			},
+		},
```

Comment:
And here, instead of hard-coding the PCI test case in the list, you'd do:

```
if pciTest != nil {
    optTCases = append(optTCases, *pciTest)
}
```